### DISCUSSION TITLE
Sonoff Basic R2 - very odd connection issues

### DISCUSSION THREAD

**Discussion by Alphapin** [arendst/Tasmota#19586] [Support request]
I am having strange connection issue with some Sonoff Basic R2 switches and have spent a whole day trying to figure it out without success, so hopefully someone can assist me.

1. Have a TP Link Deco mesh wifi network.
2. Created a 2.4ghz IoT sub network with no spaces or illegal characters in attempt to isolate the switches from the primary network.
3. I turned off 5ghz on the main network.
4. Flashed 8 Sonoff Basic R2 wifi switches, all of which were working before.
5. All switches connect to the wifi network when powered with pins connected to Windows PC via the USB and I could see them on the Deco web management app.
6. When I disconnect the switches from the pins and the USB and hook them up to regular power, only 3 of them log onto the wifi successfully.  The rest does not.
7. So I took one of the problem switch and re-flashed it (erase) with no luck.  Again it will connect to the wifi when powered through the pins and not while connected to power.  So I flashed it again using Tasmota Lite, but same thing.

So no matter what I do I can not get some of the switches to log on to the wifi when connected to power but they connect when I disconnect the power and put the pins in to power them.

Can someone give me some fresh ideas of what this can be as I am at a loss and I have done multiple searches for this problem but haven´t found anything.

/Magnus

**Comment by Alphapin** [comment]
I thought I add that these Sonoff switches have been running without any problems on another firmware for about 3 years, so they are not bad or defected.  These problems only appeared after I flashed them with Tasmota.  

Also, the firmware on the previously was not the one that they shipped with.  I flashed them all before myself.

**Reply by sfromis** [reply]
Running with other firmware does in no way prove "not bad". Tasmota has a lot of functions and may use a bit more power. And the Sonoff Basic R2 is well-known as barely able to power itself, only getting worse with age.
